.github/workflows/auto-version-bump-on-staging.yml

```
@@ -0,0 +1,75 @@
+
+jobs:
+  version_bump:
+    if: github.event.head_commit.message == 'Merge pull request' # Only run on actual merge commits
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write # Required for creating pull requests
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)